plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -91,6 +91,13 @@ data DefaultFun
     -- types, hence we include the name of the data type as a suffix.
     | ChooseData
     | ChooseUnit
+    -- Monomorphic constructors that we need for constructing e.g. Data, since polymorphic builtin
+    -- constructors are still problematic (See note [Representable built-in functions over polymorphic built-in types])
+    | MkPairData
+    | MkNilData
+    | MkConsData
+    | MkNilPairData
+    | MkConsPairData
```

Comment:
Just to be clear, you could have a single `MkCons` if you wanted to, as it's completely safe, unlike `MkNull` or `MkPair`.

Reply:
Hm, for some reason I thought it was. I'll do that, the fewer builtins the better.

Reply:
Just copy it from the example builtins.